lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -2098,7 +2099,9 @@ static void GetHomeREPLInitFile(llvm::SmallVectorImpl<char> &init_file,
   std::string init_file_name;
 
   switch (language) {
-  // TODO: Add support for a language used with a REPL.
+  case eLanguageTypeSwift:
+    init_file_name = ".lldbinit-swift-repl";
```

Comment:
Can we abstract this in the language plugin so that we don't need Swift specific logic in generic code?

Reply:
Another possible alternative is having a "scheme" where the REPL specific init file is always called `.lldbinit-<language>-repl`.